examples/linear_model/plot_ridge_coeffs.py

```diff
@@ -1,71 +1,88 @@
 """
-==============================================================
-Plot Ridge coefficients as a function of the L2 regularization
-==============================================================
+=========================================================
+Ridge coefficients as a function of the L2 regularization
+=========================================================
 
 .. currentmodule:: sklearn.linear_model
 
-:class:`Ridge` Regression is the estimator used in this example.
-Each color in the left plot represents one different dimension of the
-coefficient vector, and this is displayed as a function of the
-regularization parameter. The right plot shows how exact the solution
-is. This example illustrates how a well defined solution is
-found by Ridge regression and how regularization affects the
-coefficients and their values. The plot on the right shows how
-the difference of the coefficients from the estimator changes
-as a function of regularization.
-
-In this example the dependent variable Y is set as a function
-of the input features: y = X*w + c. The coefficient vector w is
-randomly sampled from a normal distribution, whereas the bias term c is
-set to a constant.
-
-As alpha tends toward zero the coefficients found by Ridge
-regression stabilize towards the randomly sampled vector w.
-For big alpha (strong regularisation) the coefficients
-are smaller (eventually converging at 0) leading to a
-simpler and biased solution.
-These dependencies can be observed on the left plot.
-
-The right plot shows the mean squared error between the
-coefficients found by the model and the chosen vector w.
-Less regularised models retrieve the exact
-coefficients (error is equal to 0), stronger regularised
-models increase the error.
-
-Please note that in this example the data is non-noisy, hence
-it is possible to extract the exact coefficients.
+When a model overfits, we want to regularize it's weights so it can generalize
+better on unseen data. Regularization reduces a model's reliance on specific
+information obtained from the training samples.
 
+This example illustrates how L2 regularization affects a model's performance by
+adding a penalty term to the Loss that increases with the coefficients (β).
```

Comment:
I would propose to mention `Ridge` model before going into the loss modelisation.

I think that we can use Latex to write the equation (https://docutils.sourceforge.io/docs/ref/rst/mathematics.html).